owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/accounts/AccountUtils.java

```diff
@@ -330,6 +330,7 @@ public Account getFailedAccount() {
          * OAuth2 scope
          */
         public static final String KEY_OAUTH2_SCOPE = "oc_oauth2_scope";
+        public static final String OAUTH2_OIDC_SCOPE = "openid offline_access email profile";
```

Comment:
Same as i commented in app PR, why are these constants here if they are not used in library?